.changeset/lucky-cheetahs-relate.md

```diff
@@ -0,0 +1,7 @@
+---
+"trace-deps": minor
+"trace-pkg": minor
```

Comment:
Although this is `minor` for changesets, this is a `0.X` release which allows breaking changes.